.github/workflows/release.yml

```
@@ -0,0 +1,57 @@
+name: Release
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)